packages/event-bus-redis/tsconfig.json

```diff
@@ -0,0 +1,29 @@
+{
+  "compilerOptions": {
+    "lib": ["es2020"],
+    "target": "es2020",
```

Comment:
suggestion: should we align on that, another place it is es6 and here es2020 just to be sure we use the same naming everywhere